crates/rbuilder/src/privacy/mod.rs

```diff
@@ -0,0 +1,9 @@
+//! In some installations (eg: TDX) we may want to avoid ever exposing some information.
+//! This module contains all redacting stuff to achieve that.
+//! We try to keep this redacting always configurable so we can have the full data for development.
+pub mod error_redactor;
+pub mod relay;
+pub mod validation;
```

Comment:
This is personal preference, but I'd prefer putting `relay` error redaction logic close to the relay error, and `validation` error redaction logic close to the validation error, instead of bundling all the redaction stuff in `privacy`. 

Otherwise it 
1. creates one huge module (in future maybe crate) that depends on every other module/crate in the codebase
2. imo makes the codebase less cohesive and more difficult to navigate & maintain. imagine adding a new variant to the mev_boost `RelayError`. I would prefer to stay inside the mev_boost package to impl how to redact it, instead of needing to jump into an entirely different package.

Reply:
It feels like invading the other modules with some functionality they don't really care about.
But I don't have a strong position about this. If you answer "do it" I'll go with your version.